### PR TITLE
Add advisory for race condition in Tokio

### PR DESCRIPTION
**Pull request by Darksonn**
See https://github.com/tokio-rs/tokio/issues/3929

### CODE REVIEW

**Review by alex — APPROVED** [review]

File: crates/tokio/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[advisory]
```
Comment (alex) [inline crates/tokio/RUSTSEC-0000-0000.md:1]: Needs to be wrapped in:
```markdown
```toml
contents
\```
```

Comment (alex) [reply crates/tokio/RUSTSEC-0000-0000.md:1]: That got a bit mangled, but see https://raw.githubusercontent.com/RustSec/advisory-db/main/EXAMPLE_ADVISORY.md

**Comment by Darksonn** [comment]
Ah, I just copied the code block from the README.

**Comment by Shnatsel** [comment]
I can't recall if I implemented the `~` operator. There's a good chance I didn't, and we might have to use a more verbose version specification if I so. I'll add it this Friday now that there is a use case for it.

**Comment by Darksonn** [comment]
It's past midnight here, so I wont be able to change the tildes until tomorrow.

**Comment by Shnatsel** [comment]
Properly implementing tildes is tricky, so I don't want to rush that. I've changed the version specification to use version ranges in the meanwhile.

**Comment by Shnatsel** [comment]
@alex FYI this should have specified `>= 1.8.1` instead of `^1.8.1`, because the latter would mark 2.0.0 as vulnerable. I've fixed this on main after merging this PR.

**Comment by Shnatsel** [comment]
Apologies for the inconvenience with the tilde, and thanks for reporting the issue!

To double-check, this is the list of all versions published to crates.io, and whether they're vulnerable or not according to the current version specification, ordered by upload date:

```
0.0.0 OK
0.1.0 OK
0.1.1 OK
0.1.2 OK
0.1.3 OK
0.1.4 OK
0.1.5 OK
0.1.6 OK
0.1.7 OK
0.1.8 OK
0.1.9 OK
0.1.10 OK
0.1.11 OK
0.1.12 OK
0.1.13 OK
0.1.14 OK
0.1.15 OK
0.1.16 OK
0.1.17 OK
0.1.18 OK
0.1.19 OK
0.1.20 OK
0.1.21 OK
0.1.22 OK
0.2.0-alpha.1 OK
0.2.0-alpha.2 OK
0.2.0-alpha.3 OK
0.2.0-alpha.4 OK
0.2.0-alpha.5 OK
0.2.0-alpha.6 OK
0.2.0 OK
0.2.1 OK
0.2.2 OK
0.2.3 OK
0.2.4 OK
0.2.5 OK
0.2.6 OK
0.2.7 OK
0.2.8 OK
0.2.9 OK
0.2.10 OK
0.2.11 OK
0.2.12 OK
0.2.13 OK
0.2.14 OK
0.2.15 OK
0.2.16 OK
0.2.17 OK
0.2.18 OK
0.2.19 OK
0.2.20 OK
0.2.21 OK
0.2.22 OK
0.3.0 vulnerable
0.3.1 vulnerable
0.3.2 vulnerable
0.3.3 vulnerable
0.2.23 OK
0.3.4 vulnerable
0.3.5 vulnerable
0.2.24 OK
0.3.6 vulnerable
1.0.0 vulnerable
1.0.1 vulnerable
1.0.2 vulnerable
1.1.0 vulnerable
0.2.25 OK
0.3.7 vulnerable
1.0.3 vulnerable
1.1.1 vulnerable
1.2.0 vulnerable
1.3.0 vulnerable
1.4.0 vulnerable
1.5.0 vulnerable
1.6.0 vulnerable
1.6.1 vulnerable
1.6.2 vulnerable
1.7.0 vulnerable
1.7.1 vulnerable
1.8.0 vulnerable
1.5.1 OK
1.6.3 OK
1.7.2 OK
1.8.1 OK
```

**Pull request merged**